cdap-explore-jdbc/src/test/java/co/cask/cdap/explore/jdbc/ExplorePreparedStatementTest.java

```diff
@@ -45,9 +45,9 @@ public void executeTest() throws Exception {
                       (List<QueryResult>) Lists.<QueryResult>newArrayList())
     );
 
-
+    String ns = "ns1";
```

Comment:
yes the current explore tests were passing empty strings as namespace. The ``CharMatcher`` from  guava used to accept them. We never caught it because you can't pass an empty string as the namespace name from REST APIs